### PR TITLE
05 sept

### PR DESCRIPTION
**Pull request by Sahilchouhan20**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -38,7 +39,6 @@ def update
```
Comment (rishabhharod) [inline app/controllers/posts_controller.rb:39]: if @post.save! ,if you are using ! once the work done remove it.

File: app/views/comments/index.html.erb [hunk 2]
```diff
@@ -8,36 +8,32 @@
...
+              <div class="d-flex flex-start">
                 <%if comment.post.user.avatar.attached? %>
-                    <%= image_tag main_app.rails_blob_url(comment.post.user.avatar.attachment), style:"width:60px;
-                height:60px",class:"rounded-circle shadow-1-strong me-3"%>
-                  <%else%>
-                    <img class="rounded-circle shadow-1-strong me-3"
-                src="https://mdbcdn.b-cdn.net/img/Photos/Avatars/img%20(23).webp" alt="avatar" width="60"
-                height="60" />
+                  <%= image_tag main_app.rails_blob_url(comment.post.user.avatar.attachment), style:"width:60px;height:60px",class:"rounded-circle shadow-1-strong me-3"%>
+                <%else%>
```
Comment (rishabhharod) [inline app/views/comments/index.html.erb:14]: add the css in css file, avoid to use internal and inline css

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -9,6 +9,7 @@
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
```
Comment (rishabhharod) [inline app/views/layouts/application.html.erb:9]: why you use this :-" data-turbo-track": "reload" ?

File: app/views/likes/_button.html.erb [hunk 4]
```diff
@@ -1,7 +1,16 @@
+<style>
+  .lk{
+    border:none;
+    background-color: white;
+  }
+</style>
```
Comment (rishabhharod) [inline app/views/likes/_button.html.erb:6]: add this css in css file

File: app/views/likes/_button.html.erb [hunk 5]
```diff
@@ -1,7 +1,16 @@
...
 <% like = current_user.likes.find_by(likeable: likeable) %>
```
Comment (rishabhharod) [inline app/views/likes/_button.html.erb:7]: you should have used the query in controller or model, remove this

File: app/views/likes/_button.html.erb [hunk 6]
```diff
@@ -1,7 +1,16 @@
...
-  <%=button_to("Like", likes_path(likeable),  class:"btn btn-success",  params:{ like: {likeable_id: likeable.id,  likeable_type: likeable.class.to_s}})%>
+  <%=button_to likes_path(likeable), class:"lk", params:{ like: {likeable_id: likeable.id,  likeable_type: likeable.class.to_s}} do %>
+    <i class="fa-sharp fa-regular fa-heart"></i>
+  <%end%>
 <% else %>  
-  <%=button_to("Unlinke", like_path(like), class:"btn btn-success", method: :delete)%>
-  
+  <%=button_to like_path(like), class:"lk", method: :delete do%>
+    <i class="fa-sharp fa-solid fa-heart" style="color: #f70814;"></i>
+  <% end %>
```
Comment (rishabhharod) [inline app/views/likes/_button.html.erb:15]: use fa-heart class in css file to add color

File: app/views/posts/index.html.erb [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <%@posts.each do |post|%>
-<% if current_user.following?(post.user) || current_user == post.user %>
-<%=render partial: "posts/shows", locals:{post: post}%>
-<%end%>
+  <% if current_user.following?(post.user) || current_user == post.user %>
+    <%=render partial: "posts/shows", locals:{post: post}%>
```
Comment (rishabhharod) [inline app/views/posts/index.html.erb:3]: use single quotation instead of double

File: app/views/likes/_button.html.erb (hunk 5 above)
Comment (Sahilchouhan20) [reply app/views/likes/_button.html.erb:7]: okk i will write this in controller

File: app/views/likes/_button.html.erb (hunk 6 above)
Comment (Sahilchouhan20) [reply app/views/likes/_button.html.erb:15]: ok

File: app/views/posts/index.html.erb (hunk 7 above)
Comment (Sahilchouhan20) [reply app/views/posts/index.html.erb:3]: ok

**Pull request merged**